R/dash.R

```diff
@@ -1266,6 +1400,24 @@ Dash <- R6::R6Class(
 
     # akin to https://github.com/plotly/dash/blob/d2ebc837/dash/dash.py#L338
     # note discussion here https://github.com/plotly/dash/blob/d2ebc837/dash/dash.py#L279-L284
+    custom_index = NULL,
```

Comment:
Typo at line 965: `dire ctly`

Reply:
Typo at https://github.com/plotly/dashR/pull/175/files#diff-1881af3720f8f49fd3a44e51c9fb5a0dR1150 `accomodate`

Reply:
Typo at https://github.com/plotly/dashR/pull/175/files#diff-1881af3720f8f49fd3a44e51c9fb5a0dR1470 `styleheets`

Reply:
fixed, fixed, fixed in 49c97da5861dba089371553ccd340167944eac6f